src/tests/nativeaot/SmokeTests/Determinism/Determinism.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+
+using var baseline = File.OpenRead("baseline.object");
+using var compare = File.OpenRead("compare.object");
```

Comment:
Invoke these via reflection? Downstream, I have seen determinism issues (RyuJit-induced, since then fixed) pop up in reflection internals.